Scripts/Core/Contexts/EditorVRContext.cs

```diff
@@ -7,30 +7,47 @@
 
 namespace UnityEditor.Experimental.EditorVR.Core
 {
+    public struct ContextSettings
```

Comment:
This should be in a separate file so that IEditingContext is not dependent on EditorVRContext